### DISCUSSION TITLE
Connection string to database ?

### DISCUSSION THREAD

**Discussion by jreisam** [K4ryuu/K4-System#176] [Q&A] [answered]
First of all Thank you for the addon! great job!
Well I'm trying to connect it with my mysql database, but cant find any config file present in the release to do so. Where is it or do I have to create one?

**Comment by K4ryuu** [comment, marked as answer]
When you first start the plugin a config file generated to counterstrkesharp/configs/plugins/K4-System. There you can set the plugin settings aswell as MySQL stuff. But if you open the Wiki page of the plugin, it's all written in there.
